WalletWasabi.Fluent/Helpers/StartupHelpers/LinuxStartupHelper.cs

```diff
@@ -30,11 +30,11 @@ public static async Task AddOrRemoveDesktopFileAsync(bool runOnSystemStartup)
 					"Terminal=false",
 					"X-GNOME-Autostart-enabled=true");
 
-				await File.WriteAllTextAsync(pathToDesktopFile, fileContents).ConfigureAwait(false);
+				File.WriteAllText(PathToDesktopFile, fileContent);
```

Comment:
Async/await where possible. Is there a reason to not use the async here?

Reply:
Since we write very limited data to the file, the sync method is faster.
According to this:
https://stackoverflow.com/questions/61074203/c-sharp-performance-comparison-async-vs-non-async-text-file-io-operation-via-r

Reply:
ACK.

Reply:
According to this comment it is fixed in .NET 6 https://github.com/dotnet/runtime/issues/23196#issuecomment-874249828
Can you use async?

Reply:
Sure thing.